### DISCUSSION TITLE
Calibration graph empty after manual calibration point added

### DISCUSSION THREAD

**Discussion by nogsteedslana** [NightscoutFoundation/xDrip#2740] [Q&A]
Hi. We are using MiaoMiao with a Libre sensor. Since yesterday, whenever we try to override a calibration point, the calibration graph goes empty and the sensor stops working on Xdrip. We uninstalled Xdrip and downloaded the latest version but the issue persists. We tried it on two different devices and had the same issue. Any suggestions or possible issues you can think of?

**Comment by Navid200** [comment]
Please explain what you mean by override?
Do you mean you have engineering mode enabled?

**Reply by nogsteedslana** [reply]
Engineering mode is not enabled. It happens when you tap "Stop Sensor" and instead of stopping the sensor you select "Don't stop, just reset all calibrations." After you enter a calibration value, the calibration graph is blank and Xdrip stops communicating with the MiaoMiao.

**Reply by Navid200** [reply]
You tap on reset all calibrations.  Then, you are surprised that all calibrations are gone from the calibration graph?
Do I understand you correctly?

If yes, what do you expect to happen instead?

**Reply by nogsteedslana** [reply]
Yes, we want to start a clean calibration, that's why we deleted all of the points. But when you add a new calibration point after you've reset them all, the graph stays blank. There's not even a diagonal line anymore. And then Xdrip stops communicating with the MiaoMiao.

**Reply by Navid200** [reply]
There are some conditions for a calibration to be accepted.
This thread is also about the same: https://github.com/NightscoutFoundation/xDrip/discussions/2712

**Reply by nogsteedslana** [reply]
Thank you. I will check it out.
